### PR TITLE
send Version with every request to HealthCloud endpoint

### PR DESCRIPTION
**Pull request by Glumli**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "hc-sdk-js",
-  "version": "1.0.0",
+  "version": "0.0.0",
```
Comment (Harsimran1) [inline package.json]: is this version naming in sync with Android and IOS ? I think they are using `versionName = "0.1.0"` in android.

File: webpack.config.js [hunk 2]
```diff
@@ -29,9 +30,10 @@ module.exports = (env) => {
             new webpack.DefinePlugin({
                 'global.GENTLY': env.TARGET !== 'node',
                 NODE: env.TARGET === 'node',
+                VERSION: JSON.stringify(packageJson.version),
             }),
+            new webpack.BannerPlugin(`Version: ${JSON.stringify(packageJson.version)}`),
```
Comment (Harsimran1) [inline webpack.config.js]: Is there a way to add something like commit id in banner plugin? I think frank wanted it to know which SDK is deployed. But I assume you have to add commit before knowing commit id. I suppose we don't want to update version every time we commit.

File: package.json (hunk 1 above)
Comment (Glumli) [reply package.json]: What do you mean with version naming? The current version on iOS is 0.3.0 at the moment. I just wanted to remove the 1.0.0 for now.

File: webpack.config.js (hunk 2 above)
Comment (Glumli) [reply webpack.config.js]: We only have to update the version everytime we release. I think that's ok so far. I don't know how we want to approach Franks commit id, but I guess that's another task.

File: package.json (hunk 1 above)
Comment (Harsimran1) [reply package.json]: i thought you changed from 0.0.0 to 1.0.0. Looks good.

Comment (ibihim) [reply package.json]: why not 0.1.0???

File: src/lib/hcRequest.js [hunk 3]
```diff
@@ -34,11 +37,13 @@ const hcRequest = (type, path, {
 } = {}) => {
     let retries = 0;
 
-
     const promise = () => {
         if (authorize) {
             headers.Authorization = `Bearer ${sessionHandler.getItem('HC_Auth')}`;
         }
+        if (isHealthCloudPath(path)) {
+            headers['GC-SDK-Version'] = `JS ${VERSION}`;
```
Comment (ibihim) [inline src/lib/hcRequest.js]: Didn't this trigger an eslint warning?!

File: webpack.config.js (hunk 2 above)
Comment (ibihim) [reply webpack.config.js]: 👍

File: src/lib/hcRequest.js (hunk 3 above)
Comment (ibihim) [reply src/lib/hcRequest.js]: Maybe...?
```JavaScript
const finalHeaders = { 'GC-SDK-Version': `JS ${ VERSION } `, ...headers};
```

**Pull request merged**